dependency_support/boost/workspace.bzl

```diff
@@ -21,9 +21,9 @@ def repo():
     maybe(
         http_archive,
         name = "com_github_nelhage_rules_boost",
-        url = "https://github.com/nelhage/rules_boost/archive/1e3a69bf2d5cd10c34b74f066054cd335d033d71.tar.gz",
-        strip_prefix = "rules_boost-1e3a69bf2d5cd10c34b74f066054cd335d033d71",
-        sha256 = "b3cbdceaa95b8cfe3a69ff37f8ad0e53a77937433234f6b9a6add2eff5bde333",
+        url = "https://github.com/nelhage/rules_boost/archive/1217caae292dc9f14e8109777ba43c988cf89c5b.tar.gz",
```

Comment:
q: is that the latest version?

Reply:
This version of bazel rules for boost is from 24.05.2023. It includes boost v1.82 which is the latest version of boost available through bazel rules. Newer versions of boost rules contain mostly updates of openssl dependency which is updated very often there.